catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -295,6 +297,7 @@ function PackageCreationForm({
     const addedS3Entries: S3Entry[] = []
     const addedLocalEntries: LocalEntry[] = []
     Object.entries(files.added).forEach(([path, file]) => {
+      if (path.endsWith(FI.EMPTY_DIR_MARKER)) return
```

Comment:
maybe using some prop on `file` would be more error-proof than relying on a special file name 🤔

Reply:
like, setting the contents of a file to some gibberish like `{ bucket: '$$empty', key: `$$empty`, size: 0 }` (and making that object a constant and comparing to it)

Reply:
Thanks, that's much better. So, now I use:
```ts
export const EMPTY_DIR_MARKER = {
  bucket: '[$empty$]',
  key: '[$empty$]',
  size: 0,
}
const EMPTY_DIR_MARKER_PATH = '[$.quiltkeep$]'
```